### DISCUSSION TITLE
Ajouter des couleurs pour les embeds

### DISCUSSION THREAD

**Discussion by Pietagorh** [Crownicles/Crownicles#593] [Idées acceptées / déjà implémentées]
Mettre des couleurs d'embeds customs, pour pouvoir identifier chaque "type" d'embed (profil, item, shop, etc) au premier coup d'oeil
![Capture_d’écran_de_2021-03-28_16-20-23](https://user-images.githubusercontent.com/60353007/135641824-c1f2cc5d-79a3-4b49-9642-17fc962cca65.png)

**Comment by Bombafox** [comment]
Ce serait sympa aussi avec les fight amicaux ou pas
Ce serait + simple à différencier

**Comment by BastLast** [comment]
ça a été fait sur la commande profil, je vais extraire la suggestion pour les fight amicaux en issue car c'est une bonne idée
